src/UnitConverter/Unit/Mass/Stone.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * This file is part of the jordanbrauer/unit-converter PHP package.
+ *
+ * @copyright 2017 Jordan Brauer <jbrauer.inc@gmail.com>
+ * @license MIT
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types = 1);
+
+namespace UnitConverter\Unit\Mass;
+
+/**
+ * Stone data class.
+ *
+ * @version 1.0.0
+ * @since 1.0.0
+ * @author Jordan Brauer <jbrauer.inc@gmail.com>
+ */
+class Stone extends MassUnit
+{
+    protected function configure () : void
+    {
+        $this
+            ->setName("stone")
+
+            ->setSymbol("st")
+
+            ->setUnits(0.157473044)
```

Comment:
This is the reverse of what we are looking for. Should be `6.35029` (1 stone === 6.35029 kilograms)